python/ray/serve/tests/test_fastapi.py

```diff
@@ -664,6 +665,28 @@ def incr(self):
     assert resp.json() == [0, 0]
 
 
+@pytest.mark.parametrize(
+    "docs_path,expected_path", [(None, "/docs"), ("/documentation", "/documentation")]
+)
+def test_fastapi_docs_path(
+    serve_instance: ServeControllerClient, docs_path, expected_path
+):
```

Comment:
please also test a non-fastapi deployment, the docs path should be `None` I assume

Reply:
Added!